packages/larva-css/src/02-algorithms/a-wrapper/a-wrapper.common.inline.scss

```diff
@@ -7,4 +7,11 @@
 	margin-right: auto;
 	padding-left: $spacer-050;
 	padding-right: $spacer-050;
+}
+
+.lrv-a-wrapper-100vw\@mobile-max {
```

Comment:
This will be useful!

Reply:
Can you post about how you are using it in the #larva channel?

Reply:
@laras126 I don't think I am added to the #larva channel

Reply:
Added!